docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py

```diff
@@ -2,43 +2,58 @@
 """
 Dominant Frequency Change
 *************************
-"""
+This example provides an overview for using and understanding the `Dominant Frequency Change` check.
 
-#%%
-# Imports
-# =======
+**Structure:**
 
-import pandas as pd
-from sklearn.datasets import load_iris
-from sklearn.model_selection import train_test_split
-from deepchecks.tabular.checks.integrity import DominantFrequencyChange
-from deepchecks.tabular import Dataset
+* `What is a Dominant Frequency Change? <#what-is-a-dominant-frequency-change>`__
+* `Generate Data <#generate-data>`__
+* `Run The Check <#run-the-check>`__
+* `Define a Condition <#define-a-condition>`__
 
-#%%
-# Generating Data
-# ===============
+What is a Dominant Frequency Change?
+====================================
+Dominant Frequency Change is a data integrity test which simply checks whether dominant values have increased
+significantly between the test data and a reference data.
 
-iris = load_iris(return_X_y=False, as_frame=True)
-X = iris.data
-y = iris.target
-X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3, random_state=55)
-train_dataset = Dataset(pd.concat([X_train, y_train], axis=1), 
-            features=iris.feature_names,
-            label='target')
+This check compares the dominant values of each feature in the test data to the dominant values of the same feature in
+the reference data. If the ratio of the dominant values of the test data to the reference data is greater than a
+configurable threshold, the check fails. This threshold can be configured by specifying the `ratio_change_thres`
```

Comment:
```suggestion
threshold, the check fails. This threshold can be configured by specifying the `ratio_change_thres`
```
If you say that it's configurable in the next sentence, no need to say it in this one